src/components/hero.js

```diff
@@ -44,7 +44,7 @@ const Hero = () => (
                   value=""
                   name="EMAIL"
                   class="required email"
-                  placeholder="your email here"
+                  placeholder="Your email here"
```

Comment:
Check the Figma design, it's lowercase

```suggestion
                  placeholder="your email here"
```